### ISSUE TITLE
Could you provide a client option for combined logs?

### ISSUE THREAD

**Issue by gurumark** [amir20/dozzle#220]
Hi, 
I was wondering if you can include client option or let me know how I can combine all filtered container logs in a single file.
Thanks

**Comment by amir20** [comment]
I don't think I understand what you mean @gurumark? Are you trying to export as a file or show multiple logs in parallel? 

What problem are you trying to solve and what is your use case? Maybe if I understand it better, then I can figure out how it can be done.

**Comment by gurumark** [comment]
I am trying to find ways to continue using dozzle. I like the idea but I need to feed the combined logs to another tool so that I can analyze it. I have 100s of containers and it is not really feasible to tail them. Unfortunately, I am not allowed to ship logs to any other SaaS. 
Since this tool is already has a connection to docker, I thought it would be a small change for you to offer an option in dozzle client which will dump all the connected container logs to a flat file to be consumed by another app.
This will open up another channel for you to grow as well.

**Comment by amir20** [comment]
I see. Thanks for explaining. All sounds good but I don't think this would be easy. Currently, the logs that displayed in the browser are loaded as they arrive to docker. So let's say there is 1000 lines of logs for a container. I show the last 300. As new log entries are created, I append them to the HTML. The "export" functionality you seek would only be able to export the view visible to the browser. Meaning, it wouldn't be able to export the full logs for you to analyze it.

I created a proof of concept to see what it would look like if I wanted to do provide a true export. It ended up stalling for a minute to combine all logs and it was way too slow.  I am guessing its slow to do `docker -> docker api -> dozzle backend -> dump to file`. 

This seems like a better job for a script that you write and just dumps everything to a file. Something like `docker logs <container> > file.log`. 

Let me know if there is another way to do this with dozzle.

**Comment by gurumark** [comment]
Thanks for spending your time on this. The logs are already written in the container directories. For example, something like this will tail all the logs.
sudo tail -f /var/lib/docker/containers/*/*.log
The idea of using dozzle is that it is already doing some filtering on the docker containers. It could pick up the ones needed. You could also give an option to set the tail size.

**Comment by amir20** [comment]
Hello again. So the more I think about it, the less I think it would be the right thing to add to Dozzle. Dozzle is about real-time logs and only supports `tail -f` or paginating back in time to view more. I don't currently have any sort of batch solution to download a single large log. I focused on the use case for viewing logs for debugging instead of exporting. 

I found logs in `/var/lib/docker/containers/**/*.log` but they're in JSON and it is not consistent in different hosts. For example, on Windows and Mac they would be different. It would also require for folks to mount a volume instead of just `docker.sock`.  The only consistent way would be the API which isn't great for batch export. 

For these reason, I think it's best to use a different tool for your use case.  Sorry to not offer a better solution. I think the better solution would be write up a quick script to dump everything to a file.
